storage-manager/src/Synchronizer.cpp

```diff
@@ -434,8 +435,11 @@ void Synchronizer::process(list<string>::iterator name)
             success = true;
         }
         catch(exception &e) {
-            logger->log(LOG_CRIT, "Synchronizer::process(): error sync'ing %s opFlags=%d, got '%s'.  Retrying...", key.c_str(),
-                pending->opFlags, e.what());
+            // these are often self-resolving, so we will suppress logging it for 10 iterations, then escalate
+            // to error, then to crit
+            //if (++retryCount >= 10)
+                logger->log((retryCount < 20 ? LOG_ERR : LOG_CRIT), "Synchronizer::process(): error sync'ing %s opFlags=%d, got '%s'.  Retrying...", key.c_str(),
+                    pending->opFlags, e.what());
```

Comment:
This escalates the error level for persistent errors, rather than log everything (self-correcting or not) as a critical error.